certif/app/styles/pages/login.scss

```diff
@@ -1,11 +1,12 @@
 .login-page {
   display: flex;
   flex-direction: column;
-  width:100%;
-  min-height: 100%;
```

Comment:
Pas sûre que ça sous ie ça passe ...

Reply:
En le remettant ça semble pire, il doit y avoir un autre truc qui coince ... ie a souvent du mal avec certaines propriété flex :/ ...

Reply:
Bien vu, je me rappelle maintenant d'un commentaire dans une autre app ou il fallait le garder pour IE

Reply:
En fait le bug venait du Flex qui centrait pas bien verticalement sur IE. détail ici https://stackoverflow.com/questions/31903734/ie11-flexbox-max-width-and-marginauto . Une astuce est de mettre un min-height avec une valeur inférieur du height.